deploy/helm/sumologic/README.md

```diff
@@ -2,7 +2,7 @@
 To see all available configuration for our sub-charts, please refer to their documentation.
 
   * [Falco](https://github.com/helm/charts/tree/master/stable/falco#configuration) - All Falco properties should be prefixed with `falco.` in our values.yaml to override a property not listed below.
-  * [Prometheus Operator](https://github.com/helm/charts/tree/master/stable/prometheus-operator#configuration) - All Prometheus Operator properties should be prefixed with `prometheus-operator.` in our values.yaml to override a property not listed below.
+  * [Kube-Prometheus-Stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack#configuration) - All `kube-sprometheus-stack` properties should be prefixed with `prometheus-operator.` in our values.yaml to override a property not listed below.
```

Comment:
We also need to update the values in the table below as we document some of the respective defaults right?

Reply:
nothing has changed in the configuration as compared to an earlier configuration. Which defaults are you referring to @frankreno ?